models/gaussians/vanilla.py

```diff
@@ -120,6 +120,18 @@ def shs_rest(self):
     def num_points(self):
         return self._means.shape[0]
     @property
+    def means(self):
+        return self._means
+    @property
+    def pure_scaling(self):
+        return self._scales
+    @property
+    def pure_opacity(self):
+        return self._opacities
+    @property
+    def pure_quats(self):
+        return self._quats
+    @property
```

Comment:
This is really good, but I think to avoid confusing, we can just use `model._means``model.opacities` ... in the `export_gaussians_to_ply` fn.